### ISSUE TITLE
Add `title` attributes to sidebar ID buttons

### ISSUE THREAD

**Issue by arcanistzed** [League-of-Foundry-Developers/foundryvtt-devMode#24]
**Is your feature request related to a problem? Please describe.**
It may not be immediately clear what the buttons do and it is not very accessible if using a screen reader.

**Describe the solution you'd like**
Title attributes would be added: "Copy ID" and "Print ID"

**Comment by akrigline** [comment]
Good idea, I'll admit i'm not overly concerned about accessiblity on this module as it is an _extremely_ niche tool for non-"users." But a title attribute never hurt anyone. Ensure it's localized as well :).

**Comment by mkahvi** [comment]
With v10 these probably should be data-tooltip="blah" instead.
